index.js

```diff
@@ -86,72 +84,57 @@ Collector.prototype._notifyIfFlushed = function(callback, err) {
     setImmediate(callback, err);
 };
 
-Collector.prototype._flushSeries = function(seriesName, points, callback) {
+Collector.prototype._flushPoints = function(points, callback) {
     if (!points || points.length === 0) {
         return;
     }
     var self = this;
 
     // only send N points at a time to avoid making requests too large
-    var spliceIndex;
-    if (self._client.protocol == 'udp:' || self._client.protocol == 'line') {
-      spliceIndex = self.computePointCountToSend(points.map(function (point) {
-        return JSON.stringify(point).length;
-      }), MTU_SIZE);
-    } else {
-      spliceIndex = 50;
-    }
+    var spliceIndex = 50
```

Comment:
We were seeing some problems in the spirit of "request size too large" so we made sure we're being conservative rather than failing to send data. This is the relevant commit: https://github.com/heap/node-influx-collector/commit/67293e88f9ef5c69884e88135ac9ef955373533d . Unfortunately, I was obviously not too descriptive as to why we're doing it.

Unless you have reason to believe this is no longer needed, I think we should either keep it or be prepared that it might fail again. (And if it does fail and we have to fix it, be slightly more descriptive than me in your commit messages so we don't have to wonder about the same thing in a few months 😄 )